pkg/applicationserver/io/web/redis/registry.go

```diff
@@ -53,11 +56,8 @@ func (r WebhookRegistry) Get(ctx context.Context, ids ttnpb.ApplicationWebhookId
 // List implements WebhookRegistry.
 func (r WebhookRegistry) List(ctx context.Context, ids ttnpb.ApplicationIdentifiers, paths []string) ([]*ttnpb.ApplicationWebhook, error) {
 	var pbs []*ttnpb.ApplicationWebhook
-	k := r.Redis.Key(webhookKey, unique.ID(ctx, ids))
-	keyCmd := func(ks ...string) string {
-		return r.Redis.Key(append([]string{webhookKey, unique.ID(ctx, ids)}, ks...)...)
-	}
-	err := ttnredis.FindProtos(r.Redis, k, keyCmd).Range(func() (proto.Message, func() (bool, error)) {
+	appUID := unique.ID(ctx, ids)
+	err := ttnredis.FindProtos(r.Redis, r.Redis.Key(webhookKey, appUID), func(id string) string { return r.Redis.Key(webhookKey, appUID, id) }).Range(func() (proto.Message, func() (bool, error)) {
```

Comment:
What is the benefit of this diff, especially considering that this line is way too long?

Reply:
Consistency with other implementations.
What's wrong with splitting the call into multiple lines? No vertical space is sacrificed in this case, since you need an additional line for the (redundant) variable anyway

Reply:
>Consistency with other implementations
.
This is not a sacred rule to break other rules. This is an open source project. We write readable code. If that involves variables to give meaning to intermediary values (and also shortening the lines), then that's totally fine. The compiler optimizes that away. That was the original code.

Reply:
We write open-source code, but we also maintain it - if a change is needed in `Get` method in all registries, I want to be able to do it once and then just copy-paste the change to other registries and rename the arguments. Not reapply the same change over and over again